Editor/EditorMenu.cs

```diff
@@ -11,14 +11,12 @@ namespace AmazonGameLift.Editor
 {
```

Comment:
Question: Is this class still in use or completely moved to `Editor/Resources/EditorWindow/EditorMenu.cs`?

Reply:
If this is no longer in use, can we clean up the whole file?

Reply:
I have reviewed the old code and managed to carve out another folder of scripts, which all lead to the old plugin menu.